### PR TITLE
Adds a simple page documenting paths used in our distribution.

### LINKED ISSUE #55: Add page documenting all default paths for Stackable components
Since we removed anything specific from ADR11 we should instead add a docs page that specifies all paths that are used by the Stackable platform.

Additionally this should include information if this path can be changed and if so, what parameter controls it.

### PR DESCRIPTION
**Pull request by lfrancke**
Fixes #55

### CODE REVIEW

File: modules/ROOT/pages/paths.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+This page documents the filesystem paths our components use by default.
+
+Configuration (`/etc/stackable`)::
+Most of the Stackable products are not configured via config files but if they are you'll find their configuration in subdirectories here. One example is the agent which has its configuration at `/etc/stackable/agent`.
+
+Service Configuration (`/etc/stackable/serviceconfig`)::
+This directory contains the _materialized_ configuration for each instance of a product that is being executed by our agent.
+
+This means that every (e.g.) restart of a product, or each configuration change will result in a new subdirectory in this directory. This new subdirectory will include the full configuration used by this _run_ of a product.
```
Comment (soenkeliebau) [inline modules/ROOT/pages/paths.adoc:11]: I'm not sure I fully understadn what you are trying to express by the "(e.g.)"  ? Why the brackets?

File: modules/ROOT/pages/paths.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Binaries (`/opt/stackable/<name>`)::
+There will be a subdirectory under `/opt/stackable` for each of our operators as well as the agent (e.g. `/opt/stackable/agent`) which contains the binaries of these products.
+
+Packages (`/opt/stackable/packages`)::
```
Comment (soenkeliebau) [inline modules/ROOT/pages/paths.adoc:16]: Should we mention the "_downloads" folder separately? Also, this path is configurable via `--package-directory`, would it make sense linking to the setting that controls this
?

File: modules/ROOT/pages/paths.adoc [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+This directory contains all our downloaded and expanded packages.
+Packages are the downloaded end products (e.g. Apache NiFi).
+
+Logs (`/var/log/stackable/servicelogs/<name>`)::
+This is the default location for logs for the services you deploy (e.g. the Apache Kafka broker logs).
```
Comment (soenkeliebau) [inline modules/ROOT/pages/paths.adoc:21]: Same comment as for config dir, this is configurable via `--log-directory`..

File: modules/ROOT/pages/paths.adoc [hunk 4]
```diff
@@ -0,0 +1,21 @@
+= Default Paths
+
+This page documents the filesystem paths our components use by default.
+
+Configuration (`/etc/stackable`)::
+Most of the Stackable products are not configured via config files but if they are you'll find their configuration in subdirectories here. One example is the agent which has its configuration at `/etc/stackable/agent`.
+
+Service Configuration (`/etc/stackable/serviceconfig`)::
+This directory contains the _materialized_ configuration for each instance of a product that is being executed by our agent.
```
Comment (soenkeliebau) [inline modules/ROOT/pages/paths.adoc:9]: This is configurable via `--config-directory` maybe worth linking there for completeness sake?

**Review by soenkeliebau — DISMISSED** [review]
LGTM overall, left a few questions that can be addressed or not - I'm happy either way.

File: modules/ROOT/pages/paths.adoc (hunk 4 above)
Comment (lfrancke) [reply modules/ROOT/pages/paths.adoc:9]: I opted against a link for now

Comment (lfrancke) [reply modules/ROOT/pages/paths.adoc:9]: And changed my mind again. It's not that hard after all :)

**Review by soenkeliebau — APPROVED** [review]
LGTM (haven't tested the links, not sure if the docs are built somehow automatically?)

**Pull request merged**